js/block.js

```diff
@@ -825,13 +826,36 @@ function startDragBlock(evt){
     }
     if (dragStart === 'script'){
         origTarget.classList.add('hide');
-        dom.child(origTarget.parentElement, 'input, select').classList.remove('hide');
+        var siblingInput = dom.child(origTarget.parentElement, 'input, select');
+        if (siblingInput){
+            siblingInput.classList.remove('hide');
+        }
```

Comment:
This is actually an urgent fix for dragging non-expressions in the script area, which broke in my earlier PR.